libkbfs/folder_block_ops.go

```diff
@@ -3263,22 +3263,15 @@ func (fbo *folderBlockOps) setCachedAttr(
 func (fbo *folderBlockOps) UpdateCachedEntryAttributes(
 	ctx context.Context, lState *lockState, kmd KeyMetadata,
 	dir path, op *setAttrOp) (Node, error) {
-	childPath := dir.ChildPathNoPtr(op.Name)
-
-	// find the node for the actual change; requires looking up
-	// the child entry to get the BlockPointer, unfortunately.
```

Comment:
I think originally we didn't have `setAttrOp.File`, it was added later (commit db47e897).  Probably just forgot to streamline this code at the time.